src/sloc.rs

```diff
@@ -41,40 +49,58 @@ impl Stats {
 
     #[inline(always)]
     pub fn sloc(&self) -> f64 {
-        (self.end - self.start) as f64 + 1.
+        let sloc = if self.unit {
+            self.end - self.start
```

Comment:
ncould you add a comment to explain why there is a difference here in the way to compute sloc ?

Reply:
Ok, I've added a comment.